lnwire/accept_channel.go

```diff
@@ -98,6 +98,12 @@ type AcceptChannel struct {
 	// open.
 	ChannelType *ChannelType
 
+	// LeaseExpiry represents the absolute expiration height of a channel
+	// lease. This is a custom TLV record that will only apply when a leased
+	// channel is being opened using the script enforced lease commitment
+	// type.
+	LeaseExpiry *LeaseExpiry
+
```

Comment:
Not something we need to do right now, but thinking in the future we might want to create sort of embedded sub-structs that house all the unique params for a given channel type. This way things get less cluttered as more channel tyupes are added overtime.